tests/e2e-targetallocator/targetallocator-kubernetessd/chainsaw-test.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   creationTimestamp: null
   name: targetallocator-kubernetessd
 spec:
+  namespace: chainsaw-targetallocator-kubernetessd
```

Comment:
Switching to using the static nemaspaces for the tests that assert collector config map with TA endpoint (now it contains namespace). 

We will be able to get rid of this once we migrate to v1beta1 and use the namespace substitution provided by chainsaw.